assets/sass/modules/_progress_widget.scss

```diff
@@ -0,0 +1,47 @@
+.progress-widget {
+  margin: 10px auto;
+
+  .counter-summary {
+    font-size: 18px;
+  }
+
+  .counter-bg-overlay {
+    height: 65px;
+    background-color: #e5007d;
+    position: absolute;
+    bottom: 0;
+    width: 100%;
+    z-index: 2;
+    margin-bottom: 20px;
+  }
+
+  .counter-bg-base {
+    position: relative;
+    background-color: #106ccf;
+    display: inline-block;
+    z-index: 1;
+  }
+
+  .counter-bg-ring {
+    background: url(/assets/images/xmas/counter_transparent.png);
+    z-index: 3;
+    background-repeat: no-repeat;
+    background-size: 300px 330px;
+    height: 330px;
+    width: 299px;
```

Comment:
This might causing "pixel rounding errors", meaning that the background color "bleeds" at the edges on certain browser implementations i.e Chrome on Android.